classroom.rb

```diff
@@ -0,0 +1,8 @@
+class Classroom
+  attr_accessor :label, :student
+
```

Comment:
- [ ] - [ ] I think it would be better if you use only `getters` on the `student = [ ]` variables, as they only help you retrieve the value of an instance variable for more details you can read **[here](https://fullstackheroes.com/tutorials/ruby/getter-setter/)** or its this **[articlce](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)** i.e.:

```ruby
 attr_accessor :label
 attr_reader :students
```